docs_user/assemblies/assembly_reviewing-the-openstack-control-plane-configuration.adoc

```diff
@@ -0,0 +1,15 @@
+[id="reviewing-the-openstack-control-plane-configuration_{context}"]
+
+:context: reviewing-configuration
+
+= Reviewing the {rhos_prev_long} control plane configruation
+
+Before starting the adoption workflow, pull the configuration from the {rhos_prev_long} services and {OpenStackPreviousInstaller} on your file system to back up the configuration files. You can then use the files later, during the configuration of the adopted services, and for the record to compare and make sure nothing has been missed or misconfigured.
+
+Make sure you have pull the os-diff repository and configure according to your environment:
+link:planning.md#Configuration tooling[Configure os-diff] 
```

Comment:
Does this need to be commented out?

Reply:
@matbu Should this "planning.md" link be included in the downstream docs? 
And should this link be in the downstream docs? https://github.com/openstack-k8s-operators/os-diff/blob/main/config.yaml

CC: @jistr